tlair/src/main/scala/at/forsyte/apalache/tla/types/package.scala

```diff
@@ -132,6 +134,9 @@ package object types {
 
   import at.forsyte.apalache.tla.typecomp._
 
+  type BuilderT = TBuilderInstruction
+  type BuilderUT = TlaEx
```

Comment:
The idea is that to swap back, you just turn all the `import ...{tlaU => tla, BuilderUT => BuilderT}` into `import ...{tla, BuilderT}` and it swaps back to the scoped builder.

Reply:
This makes sense to me!

Reply:
There's [an issue](#2934) related to which of these builders is treated as the "default" (i.e. which one gets the name `tla`). If we like the unsafe builder usage, and choose to take it as the default, it will flip, `BuilderT` will become `TlaEx`, and `TBuilderInstruction` will be sth like `BuilderST` (and all these imports will just naturally look like `import ... {tla, BuilderT}`.